storage/src/database/sled_backend.rs

```diff
@@ -0,0 +1,199 @@
+use crate::database::backend::{
+    BackendIterator, BackendIteratorMode, TezedgeDatabaseBackendStore,
+    TezedgeDatabaseBackendStoreIterator,
+};
+use crate::database::error::Error;
+use crate::database::tezedge_database::{TezdegeDatabaseBackendKV, KVStoreKeyValueSchema};
+use crate::{Direction, OperationsMetaStorage, BlockMetaStorage};
+use crate::operations_meta_storage;
+use crate::block_meta_storage;
+use sled::{Config, IVec, Tree};
+use std::path::{Path};
+
+pub struct SledDBBackend {
+    db: sled::Db,
+}
+
+impl SledDBBackend {
+    pub fn new<P: AsRef<Path>>(path: P, cache: u64) -> Result<Self, Error> {
+        let db = Config::default()
+            .path(path)
+            .temporary(true)
```

Comment:
@mambisi 
what is temporary?
is it persitent?

Reply:
Persistent, i will change it